go/libraries/doltcore/doltdb/root_val.go

```diff
@@ -1055,6 +1055,36 @@ func UnionTableNames(ctx context.Context, roots ...*RootValue) ([]string, error)
 	return tblNames, nil
 }
 
+type IgnoredTables struct {
+	Ignore     []string
+	DontIgnore []string
+	Conflicts  []DoltIgnoreConflictError
+}
+
+// FilterIgnoredTables takes a slice of table names and returns two new slices: one with any table names that are specified by dolt_ignore, and the other with only those tables.
```

Comment:
(minor) Looks like this godoc comment is out of date

Reply:
Fixed.